sky/backends/backend_utils.py

```diff
@@ -1969,9 +1969,9 @@ def _update_cluster_status(
     Args:
         cluster_name: The name of the cluster.
         acquire_per_cluster_status_lock: Whether to acquire the per-cluster lock
-            before updating the status.
-        need_owner_identity_check: Whether to check the owner identity before
-            updating
+          before updating the status.
```

Comment:
(styleguide nit): if 2 spaces, reduce gap to "Args" from 4 to 2 too

Reply:
ditto : )